atc/db/wall.go

```diff
@@ -0,0 +1,91 @@
+package db
+
+import (
+	"database/sql"
+	sq "github.com/Masterminds/squirrel"
+	"time"
+)
+
+//go:generate counterfeiter . Wall
+
+type Wall interface {
+	SetMessage(string) error
+	GetMessage() (string, error)
+	SetExpiration(time.Duration) error
+	GetExpiration() (time.Time, error)
+	Clear() error
+}
+
+type wall struct {
+	conn Conn
+}
+
+func NewWall(conn Conn) Wall {
```

Comment:
For my clock idea below to work, probably something like
```suggestion
func NewWall(conn Conn, clock Clock) Wall {
```
would be called for